packages/bierzo-wallet/src/routes/payment/index.e2e.spec.ts

```diff
@@ -49,6 +51,13 @@ withChainsDescribe('E2E > Payment route', () => {
     server.close();
   });
 
+  it('should make payment and redirected to payment confirmation page', async () => {
+    await fillPaymentForm(page);
```

Comment:
Could you update this to current master? I moved `await travelToPaymentE2E(browser, page, extensionPage);` from the `beforeEach` into the tests in order to be able to have the proper wait for faucet logic.